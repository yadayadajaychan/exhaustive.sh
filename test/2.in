2VAR_A 2var2
