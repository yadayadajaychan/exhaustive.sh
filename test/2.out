VAR_A=2'd0; var2=2'd0; #10;
VAR_A=2'd0; var2=2'd1; #10;
VAR_A=2'd0; var2=2'd2; #10;
VAR_A=2'd0; var2=2'd3; #10;
VAR_A=2'd1; var2=2'd0; #10;
VAR_A=2'd1; var2=2'd1; #10;
VAR_A=2'd1; var2=2'd2; #10;
VAR_A=2'd1; var2=2'd3; #10;
VAR_A=2'd2; var2=2'd0; #10;
VAR_A=2'd2; var2=2'd1; #10;
VAR_A=2'd2; var2=2'd2; #10;
VAR_A=2'd2; var2=2'd3; #10;
VAR_A=2'd3; var2=2'd0; #10;
VAR_A=2'd3; var2=2'd1; #10;
VAR_A=2'd3; var2=2'd2; #10;
VAR_A=2'd3; var2=2'd3; #10;
